app/Enums/Subscription/SubscriptionPlan.php

```
@@ -0,0 +1,53 @@
+     * @return array{
+     *     max_owned_projects?: int|null,
+     *     max_active_tasks_per_project?: int|null,
+     *     max_members_per_project?: int|null,
+     *     max_created_meetings?: int|null,
+     *     max_api_tokens?: int|null
+     * }
+     */
+    public function limits(): array
+    {
+        /** @var array<string, mixed> $limits */
+        return $limits = config("plan-limits.{$this->value}", []);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$limits'. (undefined)

(UnusedLocalVariable)